eberron.md

```diff
@@ -23,7 +23,7 @@ Created in ages long past by ancient civilizations that have become lost history
 Ancient automatons were typically built with a spring loaded omni-directional rolling ball for mobility and the ability to morph their limbs into various shapes.  Due to this ability to morph and their magical adaptability, ancient automatons can naturally attune to a piece of magical gear that they are holding.
 
 - **Starting Wounds:** 3
-- **Move/Sprint:** 2/4
+- **Move/Sprint:** 3/2
```

Comment:
You can keep it 2/4. It's an interesting mechanic where they are slow to move, but once they start they can move a lot. The only issue I can see with it is they'll have to move/sprint pretty much all the time while others can just move.